keyboards/redragon/k552/v2/config.h

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2020 Adam Honse <calcprogrammer1@gmail.com>
+ * Copyright 2023 Stephan Reichholf <stephan@reichholf.net>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define MATRIX_UNSELECT_DRIVE_HIGH
+
+/* key matrix size */
+#define RGB_MATRIX_LED_COUNT 90
```

Comment:
this is variant specific. please split into ansi/iso

Reply:
should be OK with the restructuring to ./iso as i also did some cleaning up of config.h contents.